javascript/nanoexpress/config.yaml

```diff
@@ -2,6 +2,9 @@ framework:
   github: nanoexpress/nanoexpress
   version: 1.1
 
+environment:
+  EXPERIMENTAL_FASTCALL: 1
```

Comment:
what is this for @dalisoft ?

Reply:
It's for fix from performance drop. You can read about this bit more from [here](https://github.com/uNetworking/uWebSockets.js/releases/tag/v17.2.0)